### PR TITLE
Use List comprehension for defining containers

### PR DESCRIPTION
**Pull request by dirkmueller**
- Also avoid mkdir -p as it doesn't define the permissions
- set KEA_PIDFILE_DIR
- don't set latest tag

### CODE REVIEW

File: src/bci_build/package/postgres.py
Comment (dcermak) [inline src/bci_build/package/postgres.py:1]: Could you please make this a separate commit?

Comment (dirkmueller) [reply src/bci_build/package/postgres.py:1]: it's already in https://github.com/SUSE/BCI-dockerfile-generator/pull/2095

**Review by dcermak — APPROVED** [review]
